pkg/scheduler/util/utils_test.go

```diff
@@ -50,14 +52,15 @@ func TestSortableList(t *testing.T) {
 				Priority: &p,
 			},
 		}
-		podList.Items = append(podList.Items, pod)
+		podList = append(podList, pod)
 	}
-	podList.Sort()
-	if len(podList.Items) != 10 {
-		t.Errorf("expected length of list was 10, got: %v", len(podList.Items))
+	sort.Slice(podList, func(i, j int) bool { return higherPriority(podList[i], podList[j]) })
```

Comment:
instead of defining the function, just use the implementation here and delete the definition in line 36:

```
sort.Slice(podList, func(i, j int) bool { pod.GetPodPriority(pod1) > pod.GetPodPriority(pod2) })
```

also change ```podList``` type to ```v1.Pod``` instead of ```interface{}``` in line 40